modules/data.atmosphere/R/met2CF.ALMA.R

```diff
@@ -266,6 +266,190 @@ met2CF.PalEON <- function(in.path, in.prefix, outfolder, start_date, end_date, l
         ncdf4::nc_close(nc1)
       }  ## end loop over months
     }  ## end loop over variables
+    
+ 
+    
+    # create new coordinate dimensions based on site location lat/lon
+    nc1           <- ncdf4::nc_open(old.file)
+    tdim          <- nc1$dim[["time"]]
+    met[["time"]] <- udunits2::ud.convert(met[["time"]],"days","seconds")
+    tdim$units    <- paste0("seconds since ",year,"-01-01 00:00:00")
+    tdim$vals     <- met[["time"]]
+    tdim$len      <- length(tdim$vals)
+    lat <- ncdf4::ncdim_def(name = "latitude", units = "degrees", vals = nc1$dim[["lat"]]$vals, create_dimvar = TRUE)
+    lon <- ncdf4::ncdim_def(name = "longitude", units = "degrees", vals = nc1$dim[["lon"]]$vals, create_dimvar = TRUE)
+    time <- ncdf4::ncdim_def(name = "time", units = tdim$units, vals = tdim$vals,
+                      create_dimvar = TRUE, unlim = TRUE)
+    dim <- list(lat, lon, time)
+    cp.global.atts <- ncdf4::ncatt_get(nc = nc1, varid = 0)
+    ncdf4::nc_close(nc1)
+
+    # Open new file and fill in air_temperature
+    print(year)
+    var <- ncdf4::ncvar_def(name = "air_temperature", units = "K", dim = dim,
+                     missval = as.numeric(-9999))
+    nc2 <- ncdf4::nc_create(filename = new.file, vars = var, verbose = verbose)
+    ncdf4::ncvar_put(nc = nc2, varid = "air_temperature", vals = met[["tair"]])
+
+    # air_pressure
+    insertPmet(met[["psurf"]], nc2 = nc2, var2 = "air_pressure", units2 = "Pa", dim2 = dim,
+               verbose = verbose)
+
+    # convert CO2 to mole_fraction_of_carbon_dioxide_in_air
+    # insertPmet(nc1=nc1, var1='CO2', nc2=nc2, var2='mole_fraction_of_carbon_dioxide_in_air', units2='mole/mole', dim2=dim, conv=function(x) {
+    # x * 1e6 }, verbose=verbose)
+
+    # specific_humidity
+    insertPmet(met[["qair"]], nc2 = nc2, var2 = "specific_humidity", units2 = "kg/kg", dim2 = dim,
```

Comment:
why are these names different than the standard names?

Reply:
I think @mdietze actually wrote this function. And I think the reason is because they are not standard met drivers?

Reply:
I'm not following, specific_humidity **is** the CF standard name.

Reply:
you're right I was looking at the wrong thing (the sub-list names)